doc/profile_changes_SLE15.md

```diff
@@ -0,0 +1,402 @@
+## Main differences between SLE12 and SLE15 profiles
+
+### Product selection
+
+Starting with SLE15, all products are distributed using one medium.
+You need to choose which product to install. To do so explicitly, use the
+'product' option.
+
+#### Explicit Product Selection
+
+```xml
+<software>
+  <products config:type="list">
+    <product>SLED15</product>
```

Comment:
IIRC the product is just `SLES` or `SLED` (without the version).

Reply:
Then it should be corrected also in official documentation https://susedoc.github.io/doc-sle/develop/SLES-autoyast/single-html/#Software.Selections.base_product

Reply:
But we are using it with version also in aytests so I suppose it is correct.

https://github.com/yast/aytests-tests/blob/master/aytests/sles15.xml#L160

Reply:
Does it *really* work?

```console
# zypper se -t product
Loading repository data...
Reading installed packages...

| Name                            |
+---------------------------------+
| SLED                            |
| SLES                            |
| SLES_HPC                        |
| SLES_SAP                        |
| sle-module-basesystem           |
| sle-module-desktop-applications |
| sle-module-development-tools    |
| sle-module-server-applications  |
```

I do not think so...